execution/utility.sh

```diff
@@ -1631,6 +1631,31 @@ function delete_oai_credentials() {
   return 0
 }
 
+function is_oai_credential_used() {
+  local region="$1"; shift
+  local name="$1"; shift
+
+  local oai_status_file="$( getTempFile oai_used_XXXXXX.json)"
+  local oai_id=
+  local oai_etag=
```

Comment:
Is this required?

Reply:
Extra vars are a copy/paste - I'll clean up. At the moment, the logic needs a temp file to extract the oai_id but I could trim it to use --query instead.